packages/calcite-components/src/components/tab-nav/tab-nav.tsx

```diff
@@ -406,4 +516,55 @@ export class TabNav {
       tabTitles[this.selectedTabId].focus();
     });
   }
+
+  private getOverflowIcons(): VNode | VNode[] {
+    const { messages } = this;
+    const tabNavWidth = this.el.offsetWidth;
+
+    const tabTitles = Array.from(this.el.querySelectorAll("calcite-tab-title"));
+
+    const firstTitle = tabTitles[0].getBoundingClientRect();
+    const lastTitle = tabTitles[tabTitles.length - 1].getBoundingClientRect();
+
+    const isOverflowingRight = lastTitle.right > tabNavWidth;
+    const isOverflowingLeft = firstTitle.left < 0;
+
+    const getActionChevronDirection = (overflowDirection: string): VNode => {
+      const dirActionClass: string = overflowDirection === "right" ? CSS.arrowRight : CSS.arrowLeft;
+      const dirChevronIcon: string =
+        overflowDirection === "right" ? ICON.arrowRight : ICON.arrowLeft;
+
+      const dirText: string =
+        overflowDirection === "right" ? messages.nextTabTitles : messages.previousTabsTitles;
+
+      const dirScroll = (overflowDirection: string) =>
+        overflowDirection === "right"
+          ? this.scrollToNextTabTitles()
+          : this.scrollToPreviousTabTitles();
+
+      return (
+        <calcite-action
+          class={dirActionClass}
+          icon={dirChevronIcon}
+          onClick={() => dirScroll(overflowDirection)}
+          onKeyDown={() => dirScroll(overflowDirection)}
```

Comment:
you shouldn't need onkeydown event here if you just want this to occur on enter/space. Otherwise, this needs to check which key should call the dirScroll.